yomiage-mon/.env.example

```
@@ -0,0 +1,2 @@
+TOKEN= # Discord Token
+PREFIX= # コマンドの先頭につける記号 例: `!` `-`
```

```suggestion
PREFIX= # コマンドの先頭につける記号 例: `!` `-`

```

さっきのサジェストに含まれてなかったのでこれだけ